geomstats/geometry/special_euclidean.py

```diff
@@ -617,33 +683,37 @@ def random_uniform(self, n_samples=1, point_type=None):
         ----------
         n_samples: int, optional
             default: 1
-        point_typ: str, {'vector', 'matrix'}, optional
+        point_type: str, {'vector', 'matrix'}, optional
             default: self.default_point_type
 
-
         Returns
         -------
-        random_transfo: array-like,
+        random_point: array-like,
             shape=[n_samples, {dimension, [n + 1, n + 1]}]
             an array of random elements in SE(n) having the given point_type
```

Comment:
I know this is not your line, but could you update this parameter description so that it follows the docstring guidelines? (see Contributing file).

    An array of random elements in SE(n) having the given point_type.